webpack.config.ts

```diff
@@ -41,68 +38,81 @@ class DtsBundlePlugin {
  *  limitations under the License.
  `,
         indent: '  ',
-        outputAsModuleFolder: true
+        main: './dist/src/Leanplum.d.ts',
+        name: libraryName,
+        out: '../' + this.outFileName,
+        outputAsModuleFolder: true,
+        verbose: true,
       })
+
+      this.removeSourceDefs(path.resolve(__dirname, './dist'))
     })
   }
+
+  removeSourceDefs(rootPath: string): void {
+    for (const item of fs.readdirSync(rootPath, { withFileTypes: true })) {
+      if (item.isFile() && item.name.endsWith('.d.ts') && item.name !== this.outFileName) {
+        fs.unlink(path.join(rootPath, item.name), () => { /* noop */ })
+      } else if (item.isDirectory()) {
+        this.removeSourceDefs(path.join(rootPath, item.name))
+      } else {
+        // Ignore.
```

Comment:
nit: I think it's not necessary to state it 🙂

Reply:
Removed. 🙂